safe-global/safe-wallet-web
# [Tx flow] feat: address input redesign

## What it solves

Resolves https://github.com/safe-global/safe-wallet-web/issues/2099

## How this PR fixes it
- Implements the Address component according to the new design
- Uses the modified components in the `CreateTokenTransfer`

## How to test:
1. Navigate to the Send tokens flow
2. Observe the new component design in the address inputs
  - The caret only shows if there is any option in the suggested list
  - If you are inputing an address that is not contained in your AB, it is suggested to add it

## Screenshots
<img width="541" alt="Screenshot 2023-06-15 at 15 32 46" src="https://github.com/safe-global/safe-wallet-web/assets/32431609/52ddf128-731c-48c4-b6b8-928cdbdd0c5b">

<img width="539" alt="Screenshot 2023-06-15 at 15 33 09" src="https://github.com/safe-global/safe-wallet-web/assets/32431609/93d4a20d-736b-459a-872d-650d294ad993">

<img width="590" alt="Screenshot 2023-06-15 at 17 30 11" src="https://github.com/safe-global/safe-wallet-web/assets/32431609/37a0d64b-36be-4dee-9935-de694c014f5c">

<img width="758" alt="Screenshot 2023-06-15 at 17 30 16" src="https://github.com/safe-global/safe-wallet-web/assets/32431609/fcf19335-0324-4093-9375-82b1ffc0a0de">



## Checklist
